apps/frontend/src/components/TimeCardPage/CellTypes/CommentModals/ShowCommentModal.tsx

```diff
@@ -66,21 +80,34 @@ const deleteComment = (
   if (comments.length === 1) {
     onCloseDisplay()
   }
-  // TODO: save to DB
+  var rowToMutate = row.Comment; 
+  if (rowToMutate === undefined) {
+      rowToMutate = comments;
+  }
+  // Value is null, so mark it as undefined in our processing 
+  rowToMutate = setComments([...comments, comment]);
+  console.log(rowToMutate)
+  
+  //Triggering parent class to update its references here as well 
+  updateField("Comment", rowToMutate); 
 }
 
 interface ShowCommentModalProps {
     comments: CommentSchema[];
     setComments: Function;
+    updateField: Function,
     isEditable: boolean;
     timesheetID: number;
+    row: RowSchema;
```

Comment:
Same as above - if we don't need multiple parts of the `row` data, can we only pass in the `row.Comment` part?

Reply:
e1f9579a42a756a9d442c7797c9d3b9ef29e7537